Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml

```diff
@@ -103,6 +103,15 @@
     state: plasma
   - type: Stack
     count: 1
+  - type: SolutionContainer
+    caps: None
+    contents:
+      reagents:
+      - ReagentId: Plasma
+        Quantity: 10
+  - type: Tag
+    tags:
+    - Grindable
```

Comment:
You put this on `SheetPlasma1`, which is a special prototype that only has ONE sheet of plasma in it.
This needs to be on the base prototype, `SheetPlasma`, which is used for all plasma sheets.

Reply:
O ok